docs/source/installation.rst

```diff
@@ -121,4 +121,7 @@ access the same folder.
 Build a local image
 ~~~~~~~~~~~~~~~~~~~
 
-For more information, see `ASReview LAB GitHub <https://github.com/asreview/asreview/tree/master/docker>`__.
+If you want to use a specific version of ASReview LAB, not available as
+DockerHub image, you can build your own. To build your own image, create a
+file called Dockerfile (no file extension) and fill it with the following
+code.  For more information, see `ASReview LAB GitHub <https://github.com/asreview/asreview/tree/master/docker/#build-your-own-image>`__.
```

Comment:
I think I prefer to update the text before we move it to here. The text is not very accurate and has a mismatch with our potential audience. I prefer to revert for now. 

```suggestion
For more information, see `ASReview LAB GitHub <https://github.com/asreview/asreview/tree/master/docker>`__.
```